core/src/config.rs

```diff
@@ -1,24 +1,82 @@
 use arrayref::array_ref;
 use ed25519_dalek::*;
 use failure::Error;
+use hcid::HcidEncoding;
+use lazy_static::lazy_static;
 use rand::{rngs::OsRng, Rng};
 use serde::*;
+use std::fmt;
+use std::str::FromStr;
+
 
 const SEED_SIZE: usize = 32;
 
-fn public_key_from_base64<'de, D>(deserializer: D) -> Result<PublicKey, D::Error>
-where
-    D: Deserializer<'de>,
-{
-    String::deserialize(deserializer)
-        .and_then(|s| {
-            base64::decode_config(&s, base64::STANDARD_NO_PAD)
-                .map_err(|err| de::Error::custom(err.to_string()))
-        })
-        .map(|bytes| PublicKey::from_bytes(&bytes))
-        .and_then(|maybe_key| maybe_key.map_err(|err| de::Error::custom(err.to_string())))
+
+/// Admin... keys are just ed25591 signing keys w/ some different serialization; HcAc...
+#[derive(Debug, Clone)]
+pub struct AdminPublicKey(pub PublicKey);
```

Comment:
https://github.com/Holo-Host/holo-config/pull/4#discussion_r332105183

Reply:
Again, this is to follow Rust convention; use Serialize/Deserialize so it works for *everyone*, not just us.  And for that, we need a derived type, not a type alias.  Since we can't, again by Rust convention, create Traits for types not under our control.